### PR TITLE
updated activesupport dependency declarations

### PR DESCRIPTION
**Pull request by rebelweb**
Updated activesupport dependency to allow use with rails 5

### CODE REVIEW

**Comment by phillbaker** [comment]
Thanks for the PR! Looks like the travis error is `activesupport requires Ruby version >= 2.2.2`, can you add 
```
  exclude:
    - rvm: 2.0.0
      env: ACTIVESUPPORT_VERSION=5
    - rvm: 2.1.5
      env: ACTIVESUPPORT_VERSION=5
```
to the travis file to ignore those configurations?

File: Gemfile [hunk 1]
```diff
@@ -4,11 +4,13 @@ gem 'simplecov'
...
+version = case (ENV['ACTIVESUPPORT_VERSION'] || '5')
+          when '5'
+            '>= 5.0.0.beta1'
           when '4'
             '~> 4.0'
           when '3'
             '~> 3.1'
           end
-
+
```
Comment (phillbaker) [inline Gemfile]: Extra whitespace, please remove.

**Comment by phillbaker** [comment]
Whoops, @rebelweb, sorry, I was missing the top level key of `matrix`. See for example: https://github.com/doorkeeper-gem/doorkeeper/blob/7f9bb82632f7079929d3585499269b18a347f82f/.travis.yml#L17-L20.

**Comment by rebelweb** [comment]
I got that updated and the build passing

**Comment by phillbaker** [comment]
@rebelweb can you squash your commits please?

**Comment by phillbaker** [comment]
Sorry couple more nitpicks - can you remove the trailing whitespace? And looks like there's a beta3 of rails, would you mind updating to the latest?

**Comment by rebelweb** [comment]
I have removed whitespace and updated for ActiveSupport beta3

**Comment by phillbaker** [comment]
Thanks!

**Pull request merged**